lib/src/lib/components/form-fields/select-field/field-select.model.ts

```diff
@@ -10,11 +10,13 @@ import { AbstractControl } from '@angular/forms';
 export interface FormFieldSelectOptionsFilter {
   page?: number;
   searchQuery?: string;
+  forSelectAll?: boolean;
```

Comment:
I don't quite get what this option means

Reply:
because some select are lazy loaded, we need a way to know when the user clicks on the select all to get the remaining data from the BE, but maybe that’s not a good name, suggestions?